build.gradle

```diff
@@ -9,6 +12,11 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
+if (file("vendor/protobuf").list().length == 0) {
+	throw new GradleException(
```

Comment:
I followed the convention used in this file, do you want me to fix the indentation inside the entire file?